admin/meetings/serializers.py

```diff
@@ -2,7 +2,7 @@
 
 def serialize_meeting(meeting):
     is_meeting = True
-    if hasattr(meeting, 'is_meeting'):
+    if meeting.is_meeting is not None:
```

Comment:
Are there any situations in which this would cause an exception (such as if meeting doesn't have an is_meeting attr)? Should it be `if hasattr(meeting, 'is_meeting') and meeting.is_meeting is not None:`?